### DISCUSSION TITLE
Hologram creation for players - permission to create and edit JUST own holograms

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4356] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, could you please add way to let players create holograms? Can't find plugin for it, and CMI holograms are very nice. If you can for ex. add permission to create and edit just own holograms and to specific functions there, it would be very nice.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Because players want to make their holograms before their houses, shops etc. And I can't find good plugin for it, and CMI holograms are very good I think, that I can't be scared about dropping TPS etc. with cmi holograms.

**Comment by ghost** [comment]
Imagine the performance impact with all the players having holograms. Signs are a way better alternative.

**Comment by bobhenl** [comment]
Yeah, but maybe if we can limit it per chunk, and there were plugins before on older versions, but didn't work on the new one. And it's interesting perk
